Automattic/wp-calypso
# WIP: Update/chat support titles

#### Proposed Changes

Ref p1658163453780469/1658157549.672649-slack-C03D472D64C Request from HE to not show 24/7 and 24/5 explicitly in the comparison grid.

Tooltip copy TBD.

#### Testing Instructions

- Go to /start and /plans/<site_slug>/
- You should not see 24/7 on Business/eCommerce, and not. 24/5 on Premium



#### Pre-merge Checklist

Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
